app/assets/javascripts/decidim/admin/blank_vote.js

```diff
@@ -0,0 +1,37 @@
+const PLACEHOLDER_ES = 'Voto en blanco';
```

Comment:
Hi @LauraJaimeC ,
I know that Suara now only works with spanish and catalan, but, in the future, if Suara starts to work with another languages different from these three, we will have to remember us to increase this code.
I think there are better approaching than defining the locales in the js.

Reply:
Hello @ivan-mr, I've already modified it, can you take a look now?

Thanks!